src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -122,6 +122,14 @@ MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
  */
 @property (nonatomic, readonly, nullable, copy) NSNumber * productID MTR_NEWLY_AVAILABLE;
 
+/**
+ * Network commissioning features supported by the device.
+ *
+ * The value if not nil, represents the options available in MTRNetworkCommissioningFeature type.
+ *
+ */
+@property (nonatomic, readonly, nullable, copy) NSNumber * networkCommissioningFeature MTR_NEWLY_AVAILABLE;
```

Comment:
```suggestion
@property (nonatomic, readonly, nullable, copy) NSNumber * networkCommissioningFeatures MTR_NEWLY_AVAILABLE;
```
Using a plural name would be nicer for a bit set I think.

Is there any reason we're declaring these as `nonatomic` when the getter isn't synthesized and is in fact thread-safe?

Reply:
Fixed name. I kept it nonatomic to match other properties that are backed by the same internal bits